spec/item_repository_spec.rb

```diff
@@ -0,0 +1,91 @@
+require 'item_repository'
+
+describe ItemRepository do 
+
+  def reset_table
```

Comment:
Nice way to avoid repetition here (having one method for both tables rather than two methods)